python-spec/src/somacore/data.py

```diff
@@ -45,6 +45,7 @@ def create(
         *,
         schema: pa.Schema,
         index_column_names: Sequence[str] = (options.SOMA_JOINID,),
+        domain: Optional[Sequence[Optional[Tuple[Any, Any]]]] = None,
```

Comment:
confused by this diff; all of the `domain` additions here already exists on `main`

Reply:
derp my bad i wasn't checked out to `main`

will rebase & repost, sorry